### ISSUE TITLE
Feature request: Select the copied line when copying with no text selected

### ISSUE THREAD

**Issue by NoahAndrews** [heyman/heynote#190]
**Have you searched existing Github issues to see if someone has already requested this feature?**
Yes

**Is your feature request related to a problem? Please describe.**
#177 added functionality to copy the current line when no text is selected, but it's not apparent that anything happened.

**Describe the solution you'd like**
Heynote should select the current line

**Describe alternatives you've considered**
Some kind of pop-up message could be used, I suppose

**Additional context**
The behavior I've suggested here is the same behavior used in JetBrains IDEs.

**Comment by heyman** [comment]
Hmm, when issuing a copy command with a selection, you don't get any kind of feedback either, right? You'd lose the cursor position within the line if the whole line gets selected, and maybe that's annoying?

**Comment by NoahAndrews** [comment]
That's true, but it's obvious what got copied, unlike if you had nothing selected.
